src/assets/i18n/en.json

```diff
@@ -381,7 +380,8 @@
           "empty": "Please select a port for this transport call"
         },
         "terminal": {
-          "empty": "Please select a terminal for this transport call"
+          "empty": "Please select a terminal for this port call",
+          "emptyTS": "Please select a terminal for this timestamp"
         },
```

Comment:
To my knowledge, we should no longer be asking for a terminal when creating a port call.  If so, we can avoid introducing the `emptyTS` variant and just correct the text for the original one.

Reply:
We use the `emptyTS` for timestamp creation.

Reply:
I am still not seeing why we need to distinct messages for this.  We _only_ need a terminal when we are creating a timestamp, so the message would be the same in all cases.

Reply:
Aaaaah, I misunderstood. I see what you mean, thanks.